container-tools/alpine-confd/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# ConfD Container Image
+
+# Start from the Alpine Linux base image.  (Unfortunately we can't go
```

Comment:
As ConfD support libcrypto 1.1 as of ConfD 7.2 (still have to rebuild ConfD/OTP crypto) you can remove this comment and add the following:
# Add the ConfD cryptography integration and C-library API source
ADD confd-${CONFD_VERSION}.libconfd.tar.gz /tmp

# Rebuild the CondfD crypto integration for libcrypto1.1
WORKDIR /tmp/confd-${CONFD_VERSION}/libconfd
RUN make install_crypto